.github/workflows/codenotify.yml

```
@@ -3,6 +3,10 @@ name: Codenotify
 on:
   pull_request:
     types: [opened, synchronize, ready_for_review]
+    # We're currently notifying developers only for frontend source changes.
+    # Update this glob if that ever changes and we start including backend source code changes.
+    paths:
+      - '**/frontend/src/metabase*/**'
```

Would it be easier to understand if we use 2 paths instead?

```suggestion
      - 'frontend/src/metabase*/**'
      - 'enterprise/frontend/src/metabase-enterprise/**'
```

I initially thought this won't match the enterprise path until I saw the little asterisk after the  `src/metabase` portion.

---

If we go down this road, then it's not `'frontend/src/metabase*/**'` anymore, but `'frontend/src/metabase/**'`

---

Good catch. Yeah, forgot to remove that bit.